### ISSUE TITLE
khex jvm not on jitpack

### ISSUE THREAD

**Issue by ligi** [komputing/KHex#4]
it seems it was build - but not published: https://jitpack.io/com/github/komputing/khex/1.0.0-RC2/build.log

**Comment by jitpack-io** [comment]
Hi,

Any specific module that's not resolving for you? khex-common seems to be available:
https://jitpack.io/com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2.pom

**Comment by ligi** [comment]
Thanks for looking into it! I am mainly missing the jvm modules - they seem to be found but not published:

```
Found artifact: com.github.komputing.khex:khex-kotlinMultiplatform:1.0.0-RC2
Found artifact: com.github.komputing.khex:khex-common:1.0.0-RC2
Found artifact: com.github.komputing.khex:core-jvm:1.0.0-RC2
Found artifact: com.github.komputing.khex:extensions-jvm:1.0.0-RC2
Found artifact: com.github.komputing.khex:extensions-native:1.0.0-RC2
Found artifact: com.github.komputing.khex:extensions-linuxx64:1.0.0-RC2
Found artifact: com.github.komputing.khex:extensions-common:1.0.0-RC2
EXIT_CODE=0
2019-05-12T15:21:01.647384395Z
Exit code: 0

Build artifacts:
com.github.komputing.khex:khex-kotlinMultiplatform:1.0.0-RC2
com.github.komputing.khex:khex-common:1.0.0-RC2
com.github.komputing.khex:extensions-native:1.0.0-RC2

Files: 
com/github/komputing/khex/khex-kotlinMultiplatform/1.0.0-RC2
com/github/komputing/khex/khex-kotlinMultiplatform/1.0.0-RC2/khex-kotlinMultiplatform-1.0.0-RC2.module
com/github/komputing/khex/khex-kotlinMultiplatform/1.0.0-RC2/khex-kotlinMultiplatform-1.0.0-RC2.pom
com/github/komputing/khex/khex-kotlinMultiplatform/1.0.0-RC2/khex-kotlinMultiplatform-1.0.0-RC2.pom.md5
com/github/komputing/khex/khex-kotlinMultiplatform/1.0.0-RC2/khex-kotlinMultiplatform-1.0.0-RC2.pom.sha1
```

com/github/komputing/khex/khex-common/1.0.0-RC2
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2-javadoc.jar
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2-kdoc.jar
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2-sources.jar
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2.jar
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2.pom
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2.pom.md5
com/github/komputing/khex/khex-common/1.0.0-RC2/khex-common-1.0.0-RC2.pom.sha1

com/github/komputing/khex/extensions-native/1.0.0-RC2
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2-javadoc.jar
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2-kdoc.jar
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2.jar
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2.module
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2.pom
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2.pom.md5
com/github/komputing/khex/extensions-native/1.0.0-RC2/extensions-native-1.0.0-RC2.pom.sha1
```
```

**Comment by jitpack-io** [comment]
Ah, ok.
The jar file for `core` module doesn't get installed into the local Maven repository `~/.m2`. Possibly because the dokka task failed.

The artifact is reported as found because JitPack first finds a pom file in the build directory. It then looks into ~/.m2 for the jar file.

**Comment by RiccardoM** [comment]
It seems to be a problem with Kotlin multiplatform. It fails the first time but later executions do not produce the error. 

I see the same error while running `publishToMavenLocal` on my PC, but a second execution does not trigger the error :thinking:

**Comment by RiccardoM** [comment]
PR that tries to solve the error: https://github.com/komputing/KHex/pull/6

**Comment by ligi** [comment]
@jitpack-io thanks for your help! The problem is now solved with #6
